src/utils/__tests__/chart.spec.jsx

```diff
@@ -129,7 +133,8 @@ describe( 'getTooltipTitle', () => {
     const inDate = '2020-07-01T00:00:00.000Z'
     res = sut.getTooltipTitle( inDate, interval, dateRange, true )
     expect( res ).toContain( 'Date range: 7/14/2020 - ' )
-    expect( res ).toEqual( 'Date range: 7/14/2020 - 9/30/2020' )
+    // TODO: figure out why this doesn't work in CI
+    // expect( res ).toEqual( 'Date range: 7/14/2020 - 9/30/2020' )
```

Comment:
You want to do this in another PR or this one?

Reply:
I'd do this in another one. as far as i  could tell, I  could never get this to work in CI even when it was using momentjs.

Reply:
Sorry, i took another look at this.   Looks like some of the other cases  were  incorrect and i fixed up those   tests and updated the logic.

Reply:
and it passes locally and in CI now!